src/main/java/com/openlattice/chronicle/services/message/MessageManager.kt

```diff
@@ -0,0 +1,12 @@
+package com.openlattice.chronicle.services.message
+
+import java.util.*
+
+public interface MessageManager {
```

Comment:
`public` is the default.

```suggestion
interface MessageManager {
```